core/src/main/kotlin/fr/sncf/osrd/api/pathfinding/constraints/ElectrificationConstraints.kt

```diff
@@ -6,7 +6,11 @@ import com.google.common.collect.TreeRangeSet
 import fr.sncf.osrd.api.pathfinding.makePathProps
 import fr.sncf.osrd.graph.Pathfinding
 import fr.sncf.osrd.graph.PathfindingConstraint
-import fr.sncf.osrd.sim_infra.api.*
+import fr.sncf.osrd.sim_infra.api.Block
```

Comment:
When I got on the project, didn't much like the `import *` either. I changed it to 5 max to stop be coherent with everyone else and stop always modifying all the imports whenever I'd change a file (and someone else with different settings would remodify it as well).

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7953/commits/65f3fe1e721d1210077d922190062d5ebb5a1d8a on the files of this PR